save-cloud-charts/save-cloud/templates/sandbox-mysql-service.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if .Values.sandbox_mysql.external -}}
+apiVersion: v1
+kind: Endpoints
+metadata:
+  name: sandbox-mysql-service
```

Comment:
need to wait till https://github.com/saveourtool/save-cloud/pull/1255 is merged: we will have two schemas for a single database

Reply:
no need